app/assets/javascripts/pretty-text/engines/discourse-markdown/quotes.js.es6

```diff
@@ -132,4 +145,14 @@ export function setup(helper) {
   });
 
   helper.whiteList(['img[class=avatar]']);
+  helper.whiteList({
+    custom(tag, name, value) {
+      if (tag === 'aside' && name === 'class') {
+        const m = /^quote group\-(.+)$/.exec(value);
```

Comment:
Could probably just `return /^quote group\-(.+)$/.exec(value);` right? Or perhaps to force a boolean:

`return !!/^quote group\-(.+)$/.exec(value);`